mavros/564f935/564f935.bug

```diff
@@ -77,4 +77,4 @@ fault-codes:
   - CONFIG:ARGS
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
Not sure about this one. Is this used for operators to provide commands?

Reply:
@git-afsantos, yeah, this script is used by the operator to issue commands. This seems like behaving incorrectly rather than unresponsive to me.

Reply:
BEHAVING-INCORRECTLY is the best fit here. The operator's command is misinterpreted by the rest of the system, but the robot doesn't become unresponsive.